docker/ci/scripts/prepare_ci_tests.sh

```diff
@@ -27,22 +50,21 @@ ignore_eyes_mismatches: true
 localize_apps: true
 use_my_apps: true
 skip_seed_all: true
-
-# UI test settings and secrets
 override_dashboard: \"localhost-studio.code.org\"
 override_pegasus: \"localhost.code.org\"
 dashboard_port: 3000
 pegasus_port: 3000
-build_i18n: false
+build_i18n: true
 animations_s3_directory: animations_circle/$CI_BUILD_NUMBER
 assets_s3_directory: assets_circle/$CI_BUILD_NUMBER
 files_s3_directory: files_circle/$CI_BUILD_NUMBER
 libraries_s3_directory: libraries_circle/$CI_BUILD_NUMBER
 sources_s3_directory: sources_circle/$CI_BUILD_NUMBER
-session_store_server: 'redis://ui-tests-redis:6379/0/session'
+geocoder_redis_url: 'redis://ci-tests-redis:6379/0/geocoder'
```

Comment:
I don't think we want unit tests talking to redis. what do you think about blanking out this key in `dashboard/test/test_helper.rb`? I believe this is a direction that benefits us anyway, because it helps reduce the chances that developers will have local test failures due to the contents of their locals.yml.